api/src/stampy_chat/logging.py

```diff
@@ -0,0 +1,75 @@
+import json
+from logging import *
+from discord_webhook import DiscordWebhook
+from stampy_chat.env import LOG_LEVEL, DISCORD_LOG_LEVEL, DISCORD_LOGGING_URL
+from stampy_chat.db.session import ItemAdder
+from stampy_chat.db.models import Interaction
+
+
+class DiscordHandler(StreamHandler):
+    def emit(self, record):
+        # Ignore messages that come from non chat modules
+        if record.name.startswith('stampy_chat'):
+            return
+
+        # Ignore messages that have lower levels
+        if record.levelno < getLevelName(DISCORD_LOG_LEVEL):
+            return
+
+        self.to_discord(self.format(record))
+
+    def to_discord(self, message):
+        if not DISCORD_LOGGING_URL:
+            return
+
+        while len(message) > 2000 - 8:
+            m_section, message = message[:2000 - 8], message[2000 - 8:]
```

Comment:
would some variable name for `2000 - 8` make sense? `discord_max_len_without_buffer`?